nexus/tests/integration_tests/endpoints.rs

```diff
@@ -176,8 +176,10 @@ lazy_static! {
                 name: DEMO_DISK_NAME.clone(),
                 description: "".parse().unwrap(),
             },
-            disk_source: params::DiskSource::Blank { block_size: params::BlockSize::try_from(4096).unwrap() },
-            size: ByteCount::from_gibibytes_u32(16),
+            disk_source: params::DiskSource::Blank {
+                block_size: params::BlockSize::try_from(4096).unwrap(),
+            },
+            size: ByteCount::from_gibibytes_u32(10),
```

Comment:
If you're relying on this being 10GiB in your tests, I recommend some way of checking that in the test itself. I'm seeing a lot of comments in `test_disk_backed_by_multiple_region_sets` that look like they expect the disk / zpool size to be exactly 10 gibibytes - maybe we should pull this into a `pub const: usize DISK_SIZE_GiB = 10` ? 

If someone else writing a disk-based test modifies this (seemingly arbitrary) value tomorrow:

- Best (?) case: your test would still pass, as the size would be "within expected bounds"
- Bad case: your test would suddenly fail 
- Really bad case: your test would pass, even if it wasn't supposed to

Reply:
Got it, good suggestion - implemented in c2e713a0